src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -67,8 +68,16 @@ spec:
 
 No matter if you are using Helm Charts or GitOps and Kustomize, just put the ServiceMonitor CR next to your app and apply it in the same way. Once it's applied you can check either the _ServiceMonitors Overview_ or _ServiceMonitors Details_ dashboards, or just search for the new metrics ingested in your installation's Grafana to make sure that your containers are being scraped by the new monitoring agents.
 
-__Warning:__ The ServiceMonitor needs to be labeled with `application.giantswarm.io/team: <YOUR-TEAM-NAME>` for the Prometheus Agent to be able to discover it and start collecting metrics.
+__Warning:__ The ServiceMonitor needs to be labeled with `observability.giantswarm.io/tenant: <YOUR-TENANT-NAME>` for the metrics agent to be able to discover it and start collecting metrics. Also, please make sure that the tenant you're setting with the label does actually exist in one of the [Grafana organization CRs](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/creating-grafana-organization/).
+
+If you also want to collect logs in a similar manner, you can use [PodLogs](https://docs.giantswarm.io/tutorials/observability/data-ingestion/logs/#using-podlogs) which are the equivalent resource for log ingestion.
 
 ## ServiceMonitor vs. PodMonitor
 
 In most cases, a ServiceMonitor should cover most of your monitoring use cases but it can happen on rare occasions that a container doesn't need a service to run and it doesn't make sense to create one just for the sake of monitoring it. That's when the PodMonitor comes into action. You can find a few other examples where PodMonitor makes sense [in this discussion](https://github.com/prometheus-operator/prometheus-operator/issues/3119) in the Prometheus Operator Project.
+
+## Deprecated team label
+
+As we now support multi-tenancy for metrics, you __need__ to attribute a tenant name to your servicemonitors and podmonitors to have those routed to the adequate tenant. This means that you will have to replace the `application.giantswarm.io/team: <YOUR-TEAM-NAME>` label by the `observability.giantswarm.io/tenant: <YOUR-TENANT-NAME>` one.
+
+For now, servicemonitors and podmonitors that only have the `application.giantswarm.io/team` label are routed to the "Giant Swarm" tenant by default.
```

Comment:
```suggestion
For now, servicemonitors and podmonitors that only have the `application.giantswarm.io/team` label are routed to the "Giant Swarm" tenant by default.

```

You need an empty line at the end of the file for the "severe problems" down there